Rakefile

```
@@ -229,7 +233,7 @@ namespace :release do
     latest_release_note_title = latest_release_note.lines.first
     latest_release_note_content = latest_release_note.lines[1..-1].join
     latest_release_note_version = latest_release_note_title[/[\d.]+/]
-    if latest_release_note_version != version
+    if !dry_run? && (latest_release_note_version != version)
       raise "release note isn't written"
```

If we put `!dry_run?` here, `latest_release_note_version != version` isn't evaluated without tag push.

```suggestion
    if latest_release_note_version != version
      raise "release note isn't written" unless dry_run
```

---

fix: [6ce9642](https://github.com/groonga/groonga/pull/2324/commits/6ce9642b38f9036c343e22d27852732865b2a415) Thank you so much.